telegram/ext/_application.py

```diff
@@ -361,7 +365,7 @@ def concurrent_updates(self) -> int:
 
         .. seealso:: :wiki:`Concurrency`
         """
-        return self._concurrent_updates
+        return self._concurrent_updates.max_concurrent_updates
```

Comment:
I think it would be benefitial to add a note to the docstring stating that this is now just a shortcut to `processor.max_concurrent_updates`